elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -42,6 +42,10 @@ class RestlasticSearchClientTest extends WordSpec with Matchers with ScalaFuture
     new RestlasticSearchClient(new StaticEndpoint(new Endpoint(host, port)))
   }
 
+  override def refresh(): Unit = {
+    Await.result(restClient.refresh(index), 2.seconds)
```

Comment:
As far as I could tell, `ElasticsearchIntegrationTest.refresh()` executes a refresh on all indices on the in-memory ES.  This refresh the provided index using the client.  I thought it was a good way to test client's `refresh()` since they should expose the same behavior.  What would you name it instead?